src/EntryPoints/HookHandlers.php

```
@@ -17,18 +28,52 @@ public static function onExtensionRegistration(): void {
 		$smwgNamespacesWithSemanticLinks[WB_NS_PROPERTY] = true;
 	}
 
+	public static function onParserFirstCallInit( Parser $parser ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$parser'. (undefined)

(UnusedFormalParameter)